addons/mail/static/src/core/common/message.xml

```diff
@@ -101,7 +101,7 @@
                                                         <t t-if="showSubtypeDescription" t-out="props.messageSearch?.highlight(message.subtype_description) ?? message.subtype_description"/>
                                                     </t>
                                                 </div>
-                                                <div class="o-mail-Message-seenContainer position-absolute bottom-0">
+                                                <div class="o-mail-Message-seenContainer position-absolute">
```

Comment:
missing `overflow-hidden` in line `position-relative overflow-x-auto d-inline-block` since we have `bottom: -2px` it's always overflow so that we will have a bar
![Uploading 4596938279.jpg…]()

Reply:
![4403971719](https://github.com/user-attachments/assets/85589b29-4b95-417e-8371-081a1c9a7966)